### PR TITLE
(on|off)boarding: add wiki entries

### PR DESCRIPTION
**Pull request by tomodachi94**
**Note:** I am not a wiki administrator or bureaucrat. This is based on information from https://github.com/NixOS/nixos-wiki-infra/issues/101.

The process could change, so I've linked to a wiki project page that can be easily updated.

### CODE REVIEW

**Review by Lassulus — APPROVED** [review]

File: boarding.md [hunk 1]
```diff
@@ -15,6 +15,8 @@
   - [ ] Invite into the Moderation Team organization (via Infrastructure team)
 - Email
   - [ ] Receive email for the moderation@ alias (via Infrastructure team)
+- Wiki
+  - [ ] [Request the `moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator), if desired
```
Comment (mweinelt) [inline boarding.md]:
```suggestion
  - [ ] Request acces to the [`moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator), if desired
```

Let's only link that part.

Comment (tomodachi94) [reply boarding.md]: Definitely. Fixed in latest push.

File: boarding.md [hunk 2]
```diff
@@ -15,6 +15,8 @@
   - [ ] Invite into the Moderation Team organization (via Infrastructure team)
 - Email
   - [ ] Receive email for the moderation@ alias (via Infrastructure team)
+- Wiki
+  - [ ] Request the [`moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator), if desired
```
Comment (mweinelt) [inline boarding.md]:
```suggestion
  - [ ] Request access to the [`moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator)
```

File: boarding.md [hunk 3]
```diff
@@ -33,3 +35,5 @@
   - [ ] Rotate secrets
 - Email
   - [ ] Remove from moderation@ alias (via Infrastructure team)
+- Wiki
+  - [ ] Remove the [`moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator)
```
Comment (mweinelt) [inline boarding.md]:
```suggestion
  - [ ] Remove from the [`moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator)
```

**Pull request merged**